apps/web/modules/survey/editor/components/logic-editor-conditions.test.tsx

```
@@ -0,0 +1,109 @@
+      <LogicEditorConditions
+        conditions={initialConditions}
+        updateQuestion={updateQuestion}
+        question={questionWithLogic}
+        localSurvey={localSurvey}
+        questionIdx={questionIdx}
+        logicIdx={logicIdx}
+      />
+    );
+
+    // Find the dropdown menu trigger for the condition
+    // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
```

ESLint

[error] Definition for rule 'testing-library/no-container' was not found.

(testing-library/no-container)